**Arpit Mishra, Ph.D.**

Seattle, WA 98125

arpitpdf@gmail.com  +1-206-398-5881

https://www.linkedin.com/in/arpit-mishra-phd-810941b/

- An experienced hybrid biomedical scientist with a demonstrated history of working in a professional and academic setting for over eight years.
- Currently, working on autoimmune diseases associated with non-coding variants for their possible target genes and underlying disease mechanism.
- Prioritized approx. Eight thousand five hundred autoimmune regulatory variants by combining GWAS hits and epigenomic features such as enhancer marks, ATAC-Seq peaks.
- Identified possible target genes/genomic regions for 5600 autoimmune disease-related enhancer variants using capture Hi-C in Th1, Th2, and Th17 cells.
- Allelic activity for regulatory variants in disease-relevant primary cell types such as Th1, Th2, and Th17 using allelic MPRA
- Validated MPRA hits and Capture Hi-C regulatory variant-target gene pairs using arrayed and pooled genome editing approach in primary T cells and Jurkat T cells.
- Previously completed projects in areas ranging from protein engineering, Structure-functional studies, biophysics of small molecule-DNA interactions, genome editing, Off-target CRISPR integration, and target analysis Adenovirus-mediated CRISPR edits.

- **Key Skills:**
Benchwork: Functional genomics using ChIP seq, RNA-Seq, Sc RNA-Seq (droplet and combinatorial indexing based approach), Hi-C, Capture Hi-C, MPRA, STARR-Seq, CRISPR screens, On target off target library preparation, library sequencing using Illumina platform (Next-seq 550)
- **Computational work:**
NGS data analysis for RNA-Seq, Sc RNA-Seq, HI-C, Capture Hi-C, CRISPR off-target on target analysis, and data visualization using Shell scripting, R and Python packages, version control with GIT, (https://github.com/bioarpit1)

**Professional Experience (Recent)**

**University of Washington Seattle**

**Senior Fellow,** 2017 to Present

- Currently delineating target genes for autoimmune disease-associated regulatory variants using 3D genomics, MPRA, and genome editing
- Optimized and the implemented CRISPR-CAS9-mediated gene editing primary T helper cells.
- Generated NGS libraries for different omics assays such as ChIP-Seq, RNA-Seq, ScRNA-Seq (Split-pool), Hi-C, capture Hi-C, MPRA libraries, CRISPR off-target NGS libraries.
- Library QC using bioanalyzer and NGS run using Next-Seq 550 platform.
- Demultiplexing and data analysis using appropriate pipeline such as HiCUP, Chicago (Hi-C, capture Hi-C), DE-Seq 2 (RNA-Seq), MACS and Deeptools (ChIP-Seq), Scanpy, ScPrep, Seurat and monocle3 (ScRNA-Seq), MPRAnalyze (MPRA, STARR-seq), CRISPRESSO2 (CRISPR NGS libraries).

**Institute of Genomics &amp; Integrative biology**

**Postdoctoral Associate,** 2015-2017

**Genome editing:**

- Heterologous Purification and characterization of naturally occurring high fidelity CAS9 from _francisella novicida_ (FnCAS9)
- Optimized in vitro cleavage assay (IVC) for comparing SpCAS9 and FnCAS9
- Comparison of enzyme kinetics for cleavage using IVC and biophysical methods.
- In vivo genome editing optimization for FnCAS9 in mouse embryonic stem cells.
- Off-target comparison between FnCAS9 and Sp CAS9 using ChIP-seq.
- Supplied FPLC Purified and activity tested SpCAS9 and FnCAS9 to many labs across the institute.

**Synthetic biology:**

- investigated the effects of various ionic liquids on DNA-small molecule interaction using biophysical methods such as CD spectroscopy, UV melting, and isothermal titration calorimetry(ITC)

**Educational Background**

**Ph.D. in Bio** - **Technology** Institute of Genomics and integrative biology, New Delhi and Savitribai Phule Pune University, Pune India

**Masters (MSc) in Microbiology**–TheMaharaja Sayajirao University of baroda, Vadodara, India

**Bachelors (BSc) in Zoology**  **–** The Maharaja Sayajirao University of baroda, Vadodara, India

**Research Publications**

|
- ND Jayavelu, A Jajodia **, A Mishra** , RD Hawkins - Nature communications, 2020, Candidate silencer elements for the human and mouse genomes


- Acharya S **, Mishra A** , Paul D, Ansari AH, Azhar M, Kumar M, Rauthan R, Sharma N,Aich M, Sinha D, Sharma S, Jain S, Ray A, Jain S, Ramalingam S, Maiti S,Chakraborty D. Francisella novicida Cas9 interrogates genomic DNA with very highspecificity and can be used for mammalian genome editing. Proc Natl Acad Sci U SA. 2019 Oct 15;116(42):20959-20968. doi: 10.1073/pnas.1818461116. Epub 2019 Sep30. PubMed PMID: 31570623; PubMed Central PMCID: PMC6800334**.(co-first author)**
 |
| --- |
|
- Li C, **Mishra AS** , Gil S, Wang M, Georgakopoulou A, Papayannopoulou T, HawkinsRD, Lieber A. Targeted Integration and High-Level Transgene Expression in AAVS1Transgenic Mice after In Vivo HSC Transduction with HDAd5/35++ Vectors. Mol Ther.2019 Dec 4;27(12):2195-2212. doi: 10.1016/j.ymthe.2019.08.006. Epub 2019 Aug 19.PubMed PMID: 31494053; PubMed Central PMCID: PMC6904827.
 |
|
- **Mishra A** , Hawkins RD. Three-dimensional genome architecture and emergingtechnologies: looping in disease. Genome Med. 2017 Sep 30;9(1):87. doi:10.1186/s13073-017-0477-2. Review. PubMed PMID: 28964259; PubMed Central PMCID:PMC5623062.
 |
|
- **Mishra A** , Ekka MK, Maiti S. Influence of Ionic Liquids on Thermodynamics ofSmall Molecule-DNA Interaction: The Binding of Ethidium Bromide to Calf ThymusDNA. J Phys Chem B. 2016 Mar 17;120(10):2691-700. doi: 10.1021/acs.jpcb.5b11823.Epub 2016 Mar 4. PubMed PMID: 26907668.
 |
|
- Kumar A, Kumar S, Kumar D, **Mishra A** , Dewangan RP, Shrivastava P, RamachandranS, Taneja B. The structure of Rv3717 reveals a novel amidase from Mycobacteriumtuberculosis. Acta Crystallogr D Biol Crystallogr. 2013 Dec;69(Pt 12):2543-54.doi: 10.1107/S0907444913026371. Epub 2013 Nov 19. PubMed PMID: 24311595; PubMedCentral PMCID: PMC3852659.
 |
|
- **Mishra A,** Vij M, Kumar D, Taneja V, Mondal AK, Bothra A, Rao V, Ganguli M,Taneja B. Integration host factor of Mycobacterium tuberculosis, mIHF, compactsDNA by a bending mechanism. PLoS One. 2013 Jul 26;8(7):e69985. doi:10.1371/journal.pone.0069985. Print 2013. PubMed PMID: 23922883; PubMed CentralPMCID: PMC3724605.
 |

**Awards &amp; Fellowships**

- Senior Research Fellowship Council for Scientific and Industrial Research, India
- Junior Research Fellowship Council for Scientific and Industrial Research, India
- Best poster/oral presentation at the EMBO global exchange lecture course on &quot;structural and biophysical methods for biological macromolecules in solution&quot; held at CSIR-CCMB, Hyderabad India 2012
- Participated in the prestigious STEP(science teaching experience for postdoctoral) program
- Designed and taught senior year two-credit course on genome editing in disease treatment as a STEP fellow.

**Technical Proficiencies**

| **Molecular Biology Tools &amp; Techniques** | PCR | Primer/probe design | quantitative real-time PCR (qRT-PCR) | CRISPR-Cas9 | Gene editing analysis | DNA, RNA and protein isolation, plasmid library preparation |Gibson assembly of high-throughput MPRA and CROP-Seq libraries | In vitro transcription | Gene cloning | Site-directed mutagenesis | vector design using Benchling tool|| ChIP-Seq libraries | Hi-C and Capture Hi-C libraries | ATAC-Seq libraries | RNA-Seq libraries | Single-cell RNA-Seq libraries (combinatorial indexing based)| lentiviral packaging |
| **Protein Analysis** | Recombinant protein expression and purification in _E. coli_, animal cells | western blots | Immunoprecipitation | EMSA | FPLC | CD spectroscopy | ITC titrations |
| **Bioinformatics and NGS analysis** | ChIP-Seq | ATAC-Seq |RNA-Seq |Sc RNA-Seq | Hi-C, Capture Hi-C |MPRA data analysis using R and Python packages from Bioconductor and Bioconda repositories| Input data preparation for a variety of R/Python packages using Awk, pandas, and numpy |reproductive pipeline environment stored using Jupyter notebooks and pushed them to the git repository |
| **Mammalian Cell Culture Assays** | Mammalian cell culture (representing adherent cell lines such as HEK 293-T, mESCs, Hela, and suspension cell lines such as K562, Jurkat, primary T cell culture) | PBMC to CD4 isolation| T cell activation/proliferation |Genome-editing in naïve CD4 T Cells | electroporation using NEON system| transduction using lentiviral and adenoviral particles| Stable cell line generation using lentivirus | Flow cytometry | Cryopreservation |
